CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Change Log
 -----
 
 * Make `hooks.django.DjangoApp.manage_py` method return command output.
+  (Dropped support for Python 2.6)
```

Comment:
No, I mean, make it a top level item. This is probably the most important thing in the changelog.

That there was a bug fix in manage_py is relatively unimportant to consumers of the changelog (I mean, what the details were, really all they want to know is it didn't work in the previous release, it does now). But removing support for a Python version is.